simpeg/directives/directives.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, inversion=None, dmisfit=None, reg=None, verbose=False, **kwar
             raise TypeError(f"'{key}' property has been removed. Please use 'verbose'.")
         self.inversion = inversion
         self.dmisfit = dmisfit
-        self.reg = reg
+        self.reg: ComboObjectiveFunction | None = reg
```

Comment:
Fair enough, I thought it was guaranteed that `self.reg` was always a Combo since it is enforced by the `inverse_problem.reg` (and default if not provided). But that's not true if we let the user input it.

Reply:
Done